### ISSUE TITLE
When installing a .tar.gz, npm is running. 

### ISSUE THREAD

**Issue by mvaled** [getsentry/sentry#2350]
I build the sdist on my box.  The produced .tar.gz, has the 'sentry/static/sentry/dist'  and vendor directories.  When copied to the server and performing a `pip install sentry-8.0.0dev0.tar.gz` it still runs `npm`.

My server is:
Ubuntu 14.04 , Linux m 2.6.32-39-pve (running over Proxmox).
Python is 2.7.6 (from the Ubuntu repo).
pip is 7.1.2 running inside a virtualenv, setuptools is 18.5.

**Comment by mitsuhiko** [comment]
This is unsettling because I cannot reproduce this on my box but I have different versions of almost all the things there.  Do you maybe have the output of what runs on the server on pip install?

**Comment by mvaled** [comment]
Since the npm is running with `-q`, I know is running by inspecting the pstree and netstat.  I'll try to reproduce it in my box that almost the same as the server and report back.

**Comment by mitsuhiko** [comment]
Ignore me. You are right. sdist runs npm. I'm investigating.

**Comment by mvaled** [comment]
This is a chunk of pstree when running pip install sentry.tar.gz:

```
$ pstree | grep -C5 npm
     |         |         |      |-firefox---54*[{firefox}]
     |         |         |      |-gconfd-2
     |         |         |      |-geoclue-master---2*[{geoclue-master}]
     |         |         |      |-glib-pacrunner---{glib-pacrunner}
     |         |         |      |-gnome-session-+-compiz---7*[{compiz}]
     |         |         |      |               |-guake-+-bash---pip---python---npm---8*[{npm}]
```

**Comment by mitsuhiko** [comment]
The issue is that for some reason our `sentry-package.json` is no longer included.  Investigating why.

**Comment by mvaled** [comment]
Humm... In my copy it is included:

```
 -rw-rw-r--    manu/manu       2054 sentry-8.0.0.dev0/package.json
 -rw-rw-r--    manu/manu        122 sentry-8.0.0.dev0/sentry-package.json
 -rwxrwxr-x    manu/manu      14445 sentry-8.0.0.dev0/setup.py
```

**Comment by mitsuhiko** [comment]
This actually is a lot more broken than i thought :( setuptools is pulling the source folder in still. I need to prune that out first and probably move the setup-package.json into the package. Will investigate.

In any case right now it only works because i ran develop first which builds the stuff in src.

**Comment by mitsuhiko** [comment]
Fixed in d93dc478563c01b76fa89b8d2cd02e1d34e4ae9b
